web/src/components/task/task-sidebar/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { TaskStatus } from 'api/typings/tasks';
+import { FINISHED_TASK_TOOLTIP_TEXT } from './finish-button/utils';
+
+export const getSaveButtonTooltipContent = (
+    isSaveButtonDisabled: boolean,
+    taskStatus?: TaskStatus
+) => {
+    if (isSaveButtonDisabled && taskStatus === 'Finished') {
```

Comment:
Do we need to check "isSaveButtonDisabled" here and in line 10? Is it enough to just check statuses here?

Reply:
I wanted to make sure the tooltip will appear only in the case the button is disabled

Reply:
Changes addressed